plugins/modules/network_settings_workflow_manager.py

```diff
@@ -820,6 +820,26 @@ def validate_input(self):
             }
         }
 
+        invalid_params_type = []
+
+        for config_item in self.config:
+            ip_pool = config_item.get("global_pool_details", {}).get("settings", {}).get("ip_pool", [])
+
+            for pool in ip_pool:
+                # Check for 'dhcp_server_ips'
+                if not isinstance(pool["dhcp_server_ips"], list):
+                    invalid_params_type.append("dhcp_server_ips should be a list.")
+
+                # Check for 'dns_server_ips'
+                elif not isinstance(pool["dns_server_ips"], list):
```

Comment:
if you intend to test both the dhcp_server_ip and dns_server_ip then it should not be "elif"... Can you replace elif with if?